### PR TITLE
feat: add examples for color interpolation in gradients

### PR DESCRIPTION
**Pull request by bsmth**
### Description

This PR adds an example showing how to interpolate in different color spaces in repeating gradient functions.

__Additions:__
* Example color interpolation in different color spaces in repeating gradients.

__Removals:__
* Remove non-visible fallback CSS. This can be confusing if someone visits a page using a browser that doesn't support a feature, uses the fallback and the feature appears to be working. We should prefer to fail transparently or use `@supports` with a warning / note in the example.

__Bugzilla:__

- [bugzilla.mozilla.org/show_bug.cgi?id=1861363](https://bugzilla.mozilla.org/show_bug.cgi?id=1861363)

__Related issues and pull requests:__

- [x] Parent issue https://github.com/mdn/content/issues/33520
- [x] Relnote https://github.com/mdn/content/pull/34037
- [x] BCD: https://github.com/mdn/browser-compat-data/pull/23246

### CODE REVIEW

File: files/en-us/web/css/color-interpolation-method/index.md [hunk 1]
```diff
@@ -129,6 +86,64 @@ The following example shows the effect of using different interpolation color sp
...
+    cornflowerblue 50px,
+    burlywood 100px
+  );
+}
+```
+
+#### Result
+
+{{EmbedLiveSample("hue_interpolation_in_repeating_gradients", "100%", 250)}}
+
```
Comment (estelle) [inline files/en-us/web/css/color-interpolation-method/index.md:146]:
```suggestion

The difference in colors between the 2nd and 3rd boxes demonstrates the impact of the hue-interpolation-method on color interpolation. The difference in the gradients between the first and second boxes -- note the hot pink in the first box's gradient -- demonstrates the effect of color space on color interpolation.
```
obviously not this, but something like this.

**Review by estelle — COMMENTED** [review]
It was a good example as is, but I added suggestions to make it more useful to our readers, hopefully. I suggest making the three gradients use the same color to demonstrate the effects of interpolation methods (color space and hue) within color interpolation.

Can we also remove some if not all of the browser compats. It's well supported on color-mix(), and other than the first sentence, color-mix is not otherwise included on the page.  As support for gradients is newer, we could include the bcd from the https://developer.mozilla.org/en-US/docs/Web/CSS/gradient's interpolation-color-space only.

**Comment by bsmth** [comment]
Thank you!

> Can we also remove some if not all of the browser compats. It's well supported on color-mix(), and other than the first sentence, color-mix is not otherwise included on the page. As support for gradients is newer, we could include the bcd from the https://developer.mozilla.org/en-US/docs/Web/CSS/gradient's interpolation-color-space only.

There are a lot of compats yeah, let me have a look at options for reducing the amount of space it's taking up 👀

**Comment by estelle** [comment]
does `css.types.image.gradient.interpolation_color_space` exist?

**Comment by bsmth** [comment]
> does `css.types.image.gradient.interpolation_color_space` exist?

It does not, unfortunately, it's more fine-grained at the moment, we have these (on the current `main`):

```
css.types.image.gradient.conic-gradient.interpolation_color_space
css.types.image.gradient.linear-gradient.interpolation_color_space
css.types.image.gradient.radial-gradient.interpolation_color_space
css.types.image.gradient.repeating-conic-gradient.interpolation_color_space
css.types.image.gradient.repeating-linear-gradient.interpolation_color_space
css.types.image.gradient.repeating-radial-gradient.interpolation_color_space
```

But no parent entry that covers all of them

File: files/en-us/web/css/color-interpolation-method/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/css/color-interpolation-method/index.md:141]: Added in 39d9df4dadf4d84521bb73d941e445835f7a1f47

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Comment by bsmth** [comment]
Thank you!

**Pull request merged**